src/i18n.ts

```diff
@@ -196,6 +196,8 @@ export enum I18N {
   backupFinishSuccess,
   backupFinishFinish,
   proposalTitle,
+  settingsContactEditDelete,
+  settingsContactEditSure,
```

Comment:
I meant that the enumeration should go in alphabetical order
<img width="154" alt="image" src="https://user-images.githubusercontent.com/6774813/205901300-bc6f4ef4-ae0e-45da-a292-35cbaed97666.png">
And in `en` object
<img width="284" alt="image" src="https://user-images.githubusercontent.com/6774813/205901910-1470013d-f31a-479c-b411-46778492671f.png">